packages/block-library/src/image/edit.js

```diff
@@ -333,6 +334,25 @@ export class ImageEdit extends Component {
 		}
 	}
 
+	removeNewTabRel( currentRel ) {
+		let newRel = currentRel;
+
+		if ( currentRel !== undefined && ! isEmpty( newRel ) ) {
+			if ( ! isEmpty( newRel ) ) {
+				each( NEW_TAB_REL, function( relVal ) {
+					const regExp = new RegExp( '\\s?\\b' + relVal + '\\b\\s?', 'gi' );
+					newRel = newRel.replace( regExp, '' );
```

Comment:
I think we should improve this logic for example if we set rel to `"mycustom1 noreferrer noopener"` this PR shows rel as `"mycustom1noopener"` ( not the default order needs to be manually set), what happens is that the replace removes the first noreferrer and removes all the spacing then the noopener replace does not happen.
I think we can fix this in an easy way by replacing with space instead of an empty string and then perform a trim to remove spaces added on the end or start of the line.